### PR TITLE
Run cleanup

### PR DESCRIPTION
**Pull request by shyeavr**
fix cleanup tiles remover by adding the new tile storage method and make and add script for the CLI cleanup .

### CODE REVIEW

File: Dockerfile1
Comment (dannysigalovich) [inline Dockerfile1:1]: Who is Jhon?

File: cleanup_cli/app/executers.py
Comment (dannysigalovich) [inline cleanup_cli/app/executers.py:1]: Fix the Indent please.

File: cleanup_cli/sendbox_test.py
Comment (dannysigalovich) [inline cleanup_cli/sendbox_test.py:1]: FIX indent

File: cleanup_cli/wrappers/postgres.py
Comment (dannysigalovich) [inline cleanup_cli/wrappers/postgres.py:1]: you can think of name as env param. 
if you prefer this way it okey too.

Comment (dannysigalovich) [reply cleanup_cli/wrappers/postgres.py:1]: "Orthophoto"

**Review by dannysigalovich — CHANGES_REQUESTED** [review]
Look on the comments. 
Are you sure that there is 1 exception?
(I Approve if you think its okey 👍 )

File: cleanup_cli/app/executers.py
Comment (shyeavr) [reply cleanup_cli/app/executers.py:1]: bojamoi

**Comment by shyeavr** [comment]
Finish

**Pull request closed without merging**